packages/preview-middleware-client/src/adp/controllers/AddFragment.controller.ts

```diff
@@ -296,5 +321,7 @@ export default class AddFragment extends BaseDialog {
         );
 
         await this.commandExecutor.pushAndExecuteCommand(command);
+
+        await this.rta._serializeToLrep();
```

Comment:
Wouldn't that send all unsaved changes to LREP, not just the fragment change?

Reply:
Yes, but that is the idea, before that we are showing a warning message to the user that all his unsaved changes will be saved.